src/lib/rc/rc_tests/RCTest.cpp

```diff
@@ -359,7 +359,7 @@ bool RCTest::sbus2Test()
 	bool sbus_frame_drop;
 	uint16_t max_channels = sizeof(rc_values) / sizeof(rc_values[0]);
 
-	int rate_limiter = 0;
+//	int rate_limiter = 0;
```

Comment:
Can you remove this instead of commenting it out?

Reply:
removed it